code/modules/mob/interactive.dm

```diff
@@ -126,42 +130,64 @@
 	MYPDA.name = "PDA-[real_name] ([myjob.title])"
 	equip_to_slot_or_del(MYPDA, slot_belt)
 	zone_selected = "chest"
-	if(prob(10)) //my x is augmented
-		//arms
-		if(prob((FUZZY_CHANCE_LOW+FUZZY_CHANCE_HIGH)/2))
-			var/obj/item/organ/limb/r_arm/R = locate(/obj/item/organ/limb/r_arm) in organs
-			qdel(R)
-			organs += new /obj/item/organ/limb/robot/r_arm
-		else
-			var/obj/item/organ/limb/l_arm/L = locate(/obj/item/organ/limb/l_arm) in organs
-			qdel(L)
-			organs += new /obj/item/organ/limb/robot/l_arm
-		//legs
-		if(prob((FUZZY_CHANCE_LOW+FUZZY_CHANCE_HIGH)/2))
-			var/obj/item/organ/limb/r_leg/R = locate(/obj/item/organ/limb/r_leg) in organs
-			qdel(R)
-			organs += new /obj/item/organ/limb/robot/r_leg
-		else
-			var/obj/item/organ/limb/l_leg/L = locate(/obj/item/organ/limb/l_leg) in organs
-			qdel(L)
-			organs += new /obj/item/organ/limb/robot/l_leg
-		//chest and head
-		if(prob((FUZZY_CHANCE_LOW+FUZZY_CHANCE_HIGH)/2))
-			var/obj/item/organ/limb/chest/R = locate(/obj/item/organ/limb/chest) in organs
-			qdel(R)
-			organs += new /obj/item/organ/limb/robot/chest
-		else
-			var/obj/item/organ/limb/head/L = locate(/obj/item/organ/limb/head) in organs
-			qdel(L)
-			organs += new /obj/item/organ/limb/robot/head
-		for(var/obj/item/organ/limb/LIMB in organs)
-			LIMB.owner = src
+	//arms
+	if(prob((FUZZY_CHANCE_LOW+FUZZY_CHANCE_HIGH)/2))
+		var/obj/item/organ/limb/r_arm/R = locate(/obj/item/organ/limb/r_arm) in organs
+		qdel(R)
+		organs += new /obj/item/organ/limb/robot/r_arm
+	else
+		var/obj/item/organ/limb/l_arm/L = locate(/obj/item/organ/limb/l_arm) in organs
+		qdel(L)
+		organs += new /obj/item/organ/limb/robot/l_arm
+	//legs
+	if(prob((FUZZY_CHANCE_LOW+FUZZY_CHANCE_HIGH)/2))
+		var/obj/item/organ/limb/r_leg/R = locate(/obj/item/organ/limb/r_leg) in organs
+		qdel(R)
+		organs += new /obj/item/organ/limb/robot/r_leg
+	else
+		var/obj/item/organ/limb/l_leg/L = locate(/obj/item/organ/limb/l_leg) in organs
+		qdel(L)
+		organs += new /obj/item/organ/limb/robot/l_leg
+	//chest and head
+	if(prob((FUZZY_CHANCE_LOW+FUZZY_CHANCE_HIGH)/2))
+		var/obj/item/organ/limb/chest/R = locate(/obj/item/organ/limb/chest) in organs
+		qdel(R)
+		organs += new /obj/item/organ/limb/robot/chest
+	else
+		var/obj/item/organ/limb/head/L = locate(/obj/item/organ/limb/head) in organs
+		qdel(L)
+		organs += new /obj/item/organ/limb/robot/head

```

Comment:
I can't really think of one, besides some complex associative list bullcrap which I think in the end would be more shitcodey than just copy-paste.

If anyone has a clue on how to make it as a loop i'd appreciate it.